index.html

```diff
@@ -2,27 +2,42 @@
 <html lang="en">
 	<head>
     <meta charset="utf-8">
-		<title>Document Title</title>
-		
-		<!-- external CSS link -->
+		<title>Spirit Dunkin</title>
+
+
 		<link rel="stylesheet" href="css/reset.css">
+		<!-- <link rel="stylesheet" href="css/normalize.css"> -->
 		<link rel="stylesheet" href="css/style.css">
 	</head>
-	<body>
-
+	<body background="images/pattern1.png">
+		<header>
+			<h1>SPIRIT DUNKIN</h1>
+				<h2>Flower Shop Owner, Williamsburg</h2>
+		</header>
+		<img ID = "orchids" src="images/orchids.jpg" alt="orchids">
 		<div class="page">
-		
-			About
 
-			Spirit has been working in the flower industry for 20 years. She began her work in San Francisco and has recently moved to Williamburg in Brooklyn, NY. She recently has transitioned to a lifestyle in which she enjoys frolicking in the fields on a daily basis
+			<h3>About</h3>
 
-			Her specialty is flowers for hipster weddings. Please feel free to contact Spirit anytime if you are planning a wedding that involves teepees, quilts, or lace.
+			<p>Spirit has been working in the flower industry for 20 years. She began her work in San Francisco and has recently moved to Williamburg in Brooklyn, NY. She recently has transitioned to a lifestyle in which she enjoys frolicking in the fields on a daily basis.</p>
+			<p>Her specialty is flowers for hipster weddings. Please feel free to contact Spirit anytime if you are planning a wedding that involves teepees, quilts, or lace.</p>
 
 
-			LINKS
-			http://en.wikipedia.org/wiki/Sunflower
-
+			<h3>Photos</h3>
+				<div class="image">
+					<img ID = pics src="images/sunflowers.jpg" alt="sunflowers">
```

Comment:
the formatting you're using here is a bit weird, though it may work... 

to be consistent, your elements should look like this:
```html
<img id="pics" src="images/sunflowers.jpg" alt="sunflowers">
```

note the lowercase ID, no spaces around the `=`, and text in quotes.